tests/e2e/script_test.go

```diff
@@ -355,3 +358,19 @@ func loadScript(t *testing.T, dir string, testID string, project testgcp.GCPProj
 
 	return s
 }
+
+func isConfigConnectorObject(gvk schema.GroupVersionKind) bool {
+	if gvk.Kind == "ConfigConnector" &&
+		gvk.Group == "core.cnrm.cloud.google.com" {
+		return true
+	}
+	return false
+}
+
+func isConfigConnectorContextObject(gvk schema.GroupVersionKind) bool {
+	if gvk.Kind == "ConfigConnectorContext" &&
+		gvk.Group == "core.cnrm.cloud.google.com" {
+		return true
+	}
+	return false
+}
```

Comment:
nit: maybe we can move these to pkg/k8s?

Reply:
Will do the improvement in #1489 . I want to prioritize merging this PR as it's been hanging for several weeks.